--- acid/acid_conf.php.orig	Mon Mar  4 00:02:44 2002
+++ acid/acid_conf.php	Tue Sep 17 16:07:58 2002
@@ -9,7 +9,7 @@
  *        $foo = "c:\tmp"    [OK]
  *        $foo = "c:\tmp\"   [WRONG]
  */
-$DBlib_path = "";
+$DBlib_path = "/usr/share/pear/adodb";
 
 /* The type of underlying alert database
  * 
@@ -51,7 +51,7 @@
 /* Path to the graphing library 
  *  (Note: DO NOT include a trailing backslash after the directory)
  */
-$ChartLib_path = "";
+$ChartLib_path = "/usr/share/php/phplot";
 
 /* File format of charts ('png', 'jpeg', 'gif') */
 $chart_file_format = "png";
@@ -160,7 +160,7 @@
 $whois_cache_lifetime = 40320;
 
 /* Snort spp_portscan log file */
-$portscan_file = "";
+$portscan_file = "/var/log/snort/portscan.log";
 
 /* Event cache Auto-update
  *
